db/migrate/20161128184611_add_chatroom_ref_to_walks.rb

```diff
@@ -0,0 +1,5 @@
+class AddChatroomRefToWalks < ActiveRecord::Migration[5.0]
```

Comment:
This migration had the opposite name relative to what I'd expect and I had a moment of worry.

This migration actually Adds a *walks* ref to *chatroom* not the other way around. This almost feels absurd, but I think we should change it in the file name and class declaration or delete it and regenerate with the right info, because this is confusing.